test/math/FloatPoint3DTest.ooc

```diff
@@ -57,12 +58,81 @@ FloatPoint3DTest: class extends Fixture {
 //			FIXME: Equals interface
 //			expect(FloatSize2D parse(value), is equal to(this vector3))
 		})
```

Comment:
Well, since you're probably right about interfaces, this commented out test could be removed.